### DISCUSSION TITLE
Apparent hangs frequently

### DISCUSSION THREAD

**Discussion by szmatefy** [vladmandic/sdnext#1695] [Q&A]
Hey fellas,
After a couple of image generations, the communication between the web UI and the command line is apparently hanged mostly during image generation. Every now and then I have to restart the webui.bat. Did you notice similar symptoms? 
What to do? Thank you!

**Comment by Tillerz** [comment]
I had something like that when I had a "bad" preview generation select, see here: https://github.com/vladmandic/automatic/discussions/668
